api/v1alpha5/openstackmachine_types.go

```diff
@@ -110,6 +110,9 @@ type OpenStackMachineStatus struct {
 	// +optional
 	InstanceState *InstanceState `json:"instanceState,omitempty"`
 
+	// SecurityGroups is the list of security groups IDs associated with this machine.
+	SecurityGroups []string `json:"securityGroups,omitempty"`
```

Comment:
um... I think for new features we tend to only on latest API ?

Reply:
yeah this was just for getting a build. Like I wrote in a comment above, I couldn't get the conversion working yet so I did this.

BTW I think this whole patch will go away, I was just experimenting a bit.
Now I believe a new controller is much preferred.